tests/Pinch/TransportSpec.hs

```diff
@@ -19,7 +19,7 @@ import qualified Pinch.Internal.Builder as B
 
 data MemoryConnection = MemoryConnection
   { contents :: IORef ByteString
-  , maxChunkSize :: Int -- ^ how many bytes to maximally return for one cGetSome call
+  , _maxChunkSize :: Int -- ^ how many bytes to maximally return for one cGetSome call
```

Comment:
dito

Reply:
`contents` is used in `mGetContents`, but `maxChunkSize` was never used, so it was throwing unused-top-bindings warnings. Again, solvable in other ways, this is the easiest solution